.github/workflows/acceptance.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Git Checkout
         uses: actions/checkout@v4
       - name: Set Maven Wrapper
-        run: mvn -N wrapper:wrapper -Dmaven=3.6.3
+        run: mvn --non-recursive wrapper:wrapper -Dmaven=3.9.6
```

Comment:
I replaced `-N` with `--non-recursive` just because I didn't know what it was and figured I'd spell it out so I don't have to look it up again. The main thing I wanted to do is use a modern version of maven that supports colors.